### DISCUSSION TITLE
Cheapest host computer that isn't Raspberry Pi?

### DISCUSSION THREAD

**Discussion by dhallgb** [mKeRix/room-assistant#974] [General]
I know and love the Raspberry Pi family and have used them extensively, as well as a range of other single-board computers. I need to populate a large set of buildings with multiple Bluetooth sensors running room-assistant. The Raspberry Pi Zero 2 would be excellent, but the supply restraints and software issues are constraining.

What do others recommend as a cheap alternative?

**Comment by odhiambo** [comment]
You can install RA to any Home Assistant installation. So just focus on installing the Home Assistant wherever you can and add the RA addon- https://peyanski.com/presence-detection-using-room-assistant-home-assistant-and-raspberry-pi/
